app/src/main/AndroidManifest.xml

```diff
@@ -48,7 +48,8 @@
 
         <activity android:name=".activity.HelpActivity" />
 
-        <activity android:name=".activity.RightsActivity" />
+        <activity android:name=".activity.RightsActivity"
+            android:label="Your Rights"/>
```

Comment:
Please add a string for the label to strings.xml - Even though the actual website might not be translated.